demisto_sdk/commands/content_graph/objects/content_item.py

```diff
@@ -136,6 +144,17 @@ def summary(self, marketplace: Optional[MarketplaceVersions] = None) -> dict:
                 )
             if "name" in summary_res:
                 summary_res["name"] = data.get("name_x2") or self.name
+
+            if incident_to_alert:
+                if "name" in summary_res:
+                    summary_res["name"] = replace_incident_to_alerts(
+                        summary_res["name"]
+                    )
+                if "description" in summary_res:
+                    summary_res["description"] = replace_incident_to_alerts(
+                        summary_res.get('description', "")
```

Comment:
```suggestion
                        summary_res["description"]
```
we already know it's there because of line 153 :)